LBHackney-IT/lbh-fss-portal-frontend
# Feature/list organisations

**Description**

View a list of all organisations with Organisation, User, Status, Submitted and Action columns. User can search organisations and table is paginated.

**Test instructions**

- pull this branch 
- compare to figma designs (see below) 
- test search functionality (same functionality as was used on user table)
- test pagination functionality (same functionality as was used on user table)

Figma designs:
https://www.figma.com/file/t6fanDKvLkCHJsY85ivFTF/Services-Directory?node-id=598%3A44959

![image](https://user-images.githubusercontent.com/69238887/92911231-9db16180-f420-11ea-97ea-063eb5cdf3bd.png)


**Ticket**

https://app.clickup.com/t/7euw6g
